package.json

```diff
@@ -2,7 +2,7 @@
   "name": "expensify.cash",
   "version": "1.0.77-2",
   "author": "Expensify, Inc.",
-  "homepage": "https://expensify.cash",
+  "homepage": "https://new.expensify.com",
   "description": "Expensify.cash is the next generation of Expensify: a reimagination of payments based atop a foundation of chat.",
```

Comment:
```suggestion
  "description": "New Expensify is the next generation of Expensify: a reimagination of payments based atop a foundation of chat.",
```

Reply:
Per [this comment](https://expensify.slack.com/archives/C020EPP9B9A/p1626213080240300?thread_ts=1626209742.226900&cid=C020EPP9B9A), this change is out of scope for this PR